src/mdacli/cli.py

```diff
@@ -372,4 +366,24 @@ def setup_clients():
 
 def main():
     """Execute main CLI entry point."""
-    maincli(setup_clients())
+    # modules in MDAnalysis.analysis packages that are ignored by mdacli
+    # relevant modules used in this CLI factory
+    # hydro* are removed here because they have a different folder/file
+    # structure and need to be investigated separately
+    skip_mods = ('base', 'hydrogenbonds', 'hbonds')
+    relevant_modules = (_mod for _mod in __all__ if _mod not in skip_mods)
```

Comment:
I suggest keeping skip_mods and _relevant_modules as global values. I have already import them for some personal tests. I prefer them being global.

Reply:
I revert the change. If we want to keep this global we should also capitalize this right? BTW what about the dictionary. 

https://github.com/MDAnalysis/mdacli/blob/d89ae1b9763fd3968ca47bbea13e98448ee8c507/src/mdacli/cli.py#L39-L40

Do you also want to keep this global?

Reply:
i am now on this

Reply:
refactored